src/aoai-simulated-api/src/aoai_simulated_api/generator/openai.py

```diff
@@ -494,79 +495,120 @@ def create_chat_completion_response(
 
     if streaming:
 
+        response_id = "chatcmpl-" + nanoid.non_secure_generate(size=29)
+        words = generated_content.split(" ")
+        # determine the per-token latency to use in seconds from config
+        per_token_latency_s = context.config.latency.open_ai_chat_completions.get_value() / 1000
+
         async def send_words():
+
+            # Send preamble chunks
+            chunk_string = json.dumps(
+                {
+                    "choices": [],
+                    "created": 0,
+                    "id": "",
+                    "model": "",
+                    "object": "",
+                    "prompt_filter_results": [
+                        {
+                            "prompt_index": 0,
+                            "content_filter_results": {
+                                "hate": {"filtered": False, "severity": "safe"},
+                                "self_harm": {"filtered": False, "severity": "safe"},
+                                "sexual": {"filtered": False, "severity": "safe"},
+                                "violence": {"filtered": False, "severity": "safe"},
+                            },
+                        }
+                    ],
+                },
+                separators=(",", ":"),
+            )
+            yield "data: " + chunk_string + "\n"
```

Comment:
I think an f-string would be neater here.

```suggestion
            yield f"data: {chunk_string}\n"
```

Reply:
other code (including pre-existing) uses the same pattern, so feel free to globally ignore, or accept